CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-/eng/SourceBuild* @dotnet/source-build-internal
+/eng/SourceBuild* @dotnet/source-build
```

Comment:
No write access yet?

Reply:
I was able to add it as a required reviewer so I think everything is configured correctly.

Reply:
But the CODEOWNERS validation is failing:

```
Unknown owner on line 1: make sure the team @dotnet/source-build exists, is publicly visible, and has write access to the repository
/eng/SourceBuild* @dotnet/source-build
```

Reply:
I see now what you are referring to.  @joperezr, can you give @dotnet/source-build write access?  TIA